packages/quantic/package.json

```diff
@@ -11,28 +11,32 @@
     "url": "git+https://github.com/coveo/ui-kit"
   },
   "scripts": {
-    "test": "npm run test:lint && npm run test:typings && npm run test:unit",
-    "test:lint": "eslint force-app/main/default/lwc/ --format junit -o reports/eslint.xml",
+    "test": "npm run build:staticresources && npm run lint:check:tests && npm run validate:types && npm run test:unit",
+    "lint:check:tests": "eslint force-app/main/default/lwc/ --format junit -o reports/eslint.xml",
     "lint:fix": "eslint force-app/main/default/lwc/ --fix && eslint force-app/examples/main/lwc/ --fix",
     "fix": "npm run lint:fix",
-    "copy:staticresources": "node copy-static-resources.js",
-    "build": "npm run copy:staticresources",
-    "start": "npx rimraf .localdevserver && npm run copy:staticresources && sfdx force:source:deploy -p force-app/main && sfdx force:lightning:lwc:start --port 3334",
+    "build:staticresources": "node copy-static-resources.js",
+    "build": "npm run build:staticresources && npm run build:doc",
+    "dev": "npx rimraf .localdevserver && npm run build:staticresources && npm run dev:sfdx",
+    "dev:sfdx": "sfdx force:source:deploy -p force-app/main && sfdx force:lightning:lwc:start --port 3334",
     "test:unit": "../../node_modules/.bin/lwc-jest",
     "test:unit:debug": "lwc-jest --debug",
     "test:unit:watch": "lwc-jest --watch",
     "test:unit:coverage": "lwc-jest --coverage",
-    "test:typings": "tsc",
-    "cypress:open": "npx cypress open",
-    "cypress:test": "npx cypress run --browser chrome",
-    "cypress:test:detailed": "npx cypress run --browser chrome --headless --reporter cypress/reporters/detailed-reporter.js",
+    "validate:types": "tsc",
+    "e2e:watch": "npx cypress open",
+    "e2e": "npx cypress run --browser chrome",
+    "e2e:detailed": "npx cypress run --browser chrome --headless --reporter cypress/reporters/detailed-reporter.js",
     "setup:examples": "../../node_modules/.bin/ts-node scripts/build/deploy-community.ts",
     "deploy:main": "../../node_modules/.bin/sfdx force:source:deploy -u LWC -p force-app/main",
     "deploy:examples": "../../node_modules/.bin/sfdx force:source:deploy -u LWC -p force-app/examples",
-    "doc:generate": "mkdir -p docs/out; jsdoc -c jsdoc-config.json > docs/out/quantic-docs.json",
-    "package:create": "../../node_modules/.bin/ts-node scripts/build/create-package.ts --remove-translations",
-    "package:create:publish": "../../node_modules/.bin/ts-node scripts/build/create-package.ts --remove-translations --promote",
-    "npm:publish": "node ../../scripts/deploy/publish.mjs",
+    "build:doc": "mkdir -p docs/out; jsdoc -c jsdoc-config.json > docs/out/quantic-docs.json",
+    "publish:sfdx": "../../node_modules/.bin/ts-node scripts/build/create-package.ts --remove-translations",
+    "promote:sfdx": "npm run publish:sfdx -- --promote",
```

Comment:
I wasn't exactly sure what `package:create` and `package:create:publish` did. Do these names make sense?

Reply:
yep that's pretty good